lntest/btcd.go

```diff
@@ -12,22 +12,22 @@ import (
 // BtcdBackendConfig is an implementation of the BackendConfig interface
 // backed by a btcd node.
 type BtcdBackendConfig struct {
-	// RPCConfig houses the connection config to the backing btcd instance.
-	RPCConfig rpcclient.ConnConfig
+	// rpcConfig houses the connection config to the backing btcd instance.
+	rpcConfig rpcclient.ConnConfig
```

Comment:
Any reason not to squash this commit (https://github.com/lightningnetwork/lnd/pull/1621/commits/97aacca8100f3df22155cac947534702d926670d) with the one that introduced the struct?

Reply:
Where it was introduced the fields were accessed outside the `lntest` package, so they had to be exported. But can squash it with the commit where they are no longer accessed outside `lntest` (https://github.com/lightningnetwork/lnd/pull/1621/commits/bb16b6af794b6dc291970ba91d3039eec5748106)

Reply:
Squashed commit: https://github.com/lightningnetwork/lnd/pull/1621/commits/ed50d208392fdad95f0eed88489bfdf93f653eea